ups/ci_middleware.table

```diff
@@ -1,9 +1,10 @@
-# List EUPS dependencies of this package here.
-# - Any package whose API is used directly should be listed explicitly.
-# - Common third-party packages can be assumed to be recursively included by
-#   the "base" package.
-setupRequired(base)
+setupRequired(sconsUtils)
```

Comment:
We really need sphgeom in this list as well since it's used directly by the code here.

Reply:
Also `resources`, `afw`, `daf_base`, and  `geom` are used.